### DISCUSSION TITLE
Any chance for remark-directive to adapt remark-admonitions format (starting with :::tip pro tip)

### DISCUSSION THREAD

**Discussion by Jinjiang** [remarkjs/remark#1374] [Q&A] [answered]
I'm trying to migrate mdx v1 with remark-admonitions into mdx v3 with remark-directive.

According to https://github.com/elviswolcott/remark-admonitions?tab=readme-ov-file#usage

What i've tried so far is:

In MDX v3 with remark-directive:

```
:::tip
`remark-admonitions` is pretty great!
:::
```

![image](https://github.com/user-attachments/assets/51e60e33-afbd-4e09-9633-d2b8833eaa01)

...works, however:

```
:::tip pro tip
`remark-admonitions` is pretty great!
:::
```

![image](https://github.com/user-attachments/assets/6614e89f-a2d7-4c1e-8722-d61ee12310ae)

... doesn't.

Is there any chance we can make it?

Thanks.

**Comment by wooorm** [comment, marked as answer]
Hi! No, the syntax is different. Intentionally so. Perhaps regex for `:::(\w+) (.+)` and replace by `:::\1[\2]`.

**Reply by Jinjiang** [reply]
That's a good idea! Thanks so much.
